frontend/src/app/components/TransactionsTable.tsx

```diff
@@ -0,0 +1,81 @@
+// src/components/TransactionsTable.tsx
+
+import React from 'react';
+import { Transaction } from "@/services/transactionService";
+
+interface TransactionsTableProps {
+  transactions: Transaction[];
+}
+
+const TransactionsTable: React.FC<TransactionsTableProps> = ({ transactions }) => {
+  return (
+    <div className="overflow-x-auto shadow-md rounded-lg">
+      <table
+        className="min-w-full bg-white"
+        role="table"
+        aria-label="Recent Transactions"
+      >
+        <thead className="bg-gray-50">
```

Comment:
Could this header be light green instead of gray?

Reply:
Thank you! For sure, let's go with this version.